src/v2/Components/Artwork/Details.tsx

```diff
@@ -277,7 +280,11 @@ export const LotCloseInfo: React.FC<LotCloseInfoProps> = ({
     return null
   }
 
-  const timerCopy = getSaleOrLotTimerInfo(time, { hasStarted: saleHasStarted })
+  const timerCopy = getSaleOrLotTimerInfo(time, {
+    hasStarted: saleHasStarted,
+    lotsAreClosing: lotsAreClosing,
```

Comment:
I don't think this param needs to be passed in. Looking at the helper, https://github.com/artsy/force/blob/3e5973b611bb455d043a749992f498ce31a25550/src/v2/Utils/getSaleOrLotTimerInfo.ts#L57 is only used alongside the `isSaleInfo` param (which signifies that the timer copy is being generated for a sale, vs. for a lot), so I think you can leave this off.